docs/eventing/samples/writing-ra-source/04-reconciler.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Receive Adapter Implementation and Design"
```

Comment:
These titles seem to have been copied and pasted from page 05 on the Receive Adapter. Should be updated for this page.